ocs_ci/ocs/cluster.py

```diff
@@ -815,9 +815,9 @@ def rgw_health_check(self, count):
                 resource_count=count, timeout=timeout, sleep=3,
             )
 
-            actual_rgw = 1
+            expected_rgw = 1
```

Comment:
well,

If the expected is 1 and you will pass count=1 I'm not sure what this function does.
Don't we need to call a function to get the actual RGW pods?

Reply:
I have update get_rgw_pod function to get number of rgw pod running in the cluster and return list object